frontend/src/metabase/visualizations/shared/settings/series.ts

```diff
@@ -65,7 +69,23 @@ export const getSeriesDefaultLineMarker = (
 
 export const getSeriesDefaultLineMissing = (
   settings: ComputedVisualizationSettings,
-) => settings["line.missing"] ?? "interpolate";
+  display: CardDisplayType,
+  stackType: StackType,
+) =>
+  settings["line.missing"] ??
+  (hasInterpolatedLineMissingOption(display, stackType)
+    ? "interpolate"
+    : "zero");
```

Comment:
for stacked area charts, [the default value should be](https://linear.app/metabase/issue/UXW-128/linear-interpolation-zero-values-on-stacked-area#comment-5ba175e2) replaced with zero